plugins/trace_api_plugin/request_handler.cpp

```diff
@@ -48,7 +48,8 @@ namespace {
                ("account", a.account.to_string())
                ("action", a.action.to_string())
                ("authorization", process_authorizations(a.authorization, yield))
-               ("data", fc::to_hex(a.data.data(), a.data.size()));
+               ("data", fc::to_hex(a.data.data(), a.data.size()))
+               ("return_value", fc::to_hex(a.return_value.data(), a.return_value.size()));    // add return_value if input is action_trace_v1
```

Comment:
Similar to "data" -> "params", you should attempt to deserialize the return value and provide it deserialized.
Maybe "return_value_data" and "return_value" for the json names.